config/dexs.json

```diff
@@ -915,6 +915,12 @@
     "0xAC4c6e212A361c968F1725b4d055b47E63F80b75",
     "0xd30d8ca2e7715ee6804a287eb86fafc0839b1380"
   ],
+  "ronin": [
+    "0xFEd2370437dF9c1302Fe4f0840A91fc0091DA1F2",
+    "0xc07c6BF7b8de75c53E4E341863cfe10955Bf298b",
+    "0xAAAeD431A0101303c45598d646236d661197c889",
+    "0x26e674d797e873811f705b35fABc09b59f33fFaF"
```

Comment:
@0xDEnYO could you also whiltelist kyberswap (`0x6131b5fae19ea4f9d964eac0408e4408b66337b5`) on ronin? 😇